examples/performance/00_overview.ipynb

```diff
@@ -1216,14 +1216,14 @@
       "\n",
       "int Kernel(struct dataobj *restrict f_vec, const float h_y, struct dataobj *restrict u_vec, const int x_size, const int y0_blk0_size, const int y_size, const int z_size, const int time_M, const int time_m, const int x0_blk0_size, const int x_M, const int x_m, const int y_M, const int y_m, const int z_M, const int z_m, const int nthreads, struct profiler * timers)\n",
       "{\n",
-      "  float *r0_vec;\n",
-      "  posix_memalign((void**)&r0_vec, 64, sizeof(float[x_size][y_size][z_size]));\n",
       "  float **pr2_vec;\n",
-      "  posix_memalign((void**)&pr2_vec, 64, sizeof(float*)*nthreads);\n",
+      "  posix_memalign((void**)(&pr2_vec),64,nthreads*sizeof(float*));\n",
+      "  float *r0_vec;\n",
+      "  posix_memalign((void**)(&r0_vec),64,x_size*y_size*z_size*sizeof(float));\n",
       "  #pragma omp parallel num_threads(nthreads)\n",
       "  {\n",
       "    const int tid = omp_get_thread_num();\n",
-      "    posix_memalign((void**)&pr2_vec[tid], 64, sizeof(float[y0_blk0_size + 4][z_size]));\n",
+      "    posix_memalign((void**)(&(pr2_vec[tid])),64,z_size*(y0_blk0_size + 4)*sizeof(float));\n",
```

Comment:
This here seems correct. But is it equivalent to the previous one?
The previous one seem of a different size?

Reply:
why?